### DISCUSSION TITLE
Liquidcrystal i2c

### DISCUSSION THREAD

**Discussion by ollivtesteur** [VasilKalchev/LiquidMenu#114] [Q&A] [answered]
hello, thank you for your work which looks great.  I wonder if your library is compatible with i2c LCDs

Thanks

**Comment by skinda0** [comment, marked as answer]
Yes there is an example file in the library:
https://github.com/VasilKalchev/LiquidMenu/blob/master/examples/I_I2C_menu/I_I2C_menu.ino
